charts/opentelemetry-collector/templates/config/_agent-config-override.tpl

```diff
@@ -3,20 +3,17 @@ Default config override for agent collector deamonset
 */}}
 {{- define "opentelemetry-collector.agentConfigOverride" -}}
 exporters:
-  {{- if .Values.standaloneCollector.enabled }}
+  {{- if .Values.agentCollector.enabled }}
```

Comment:
this should be `.Values.standaloneCollector.enabled`. if a standalone collector is enabled, we send the telemetry to it.